<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="base_pose_topic" default="~base_pose"/>
    <arg name="topo_position_service" default="topological_position"/>

    <node name="robot_pose_test" pkg="topological_map_ros" type="robot_pose_test" output="screen">
        <param name="base_pose_topic" type="string" value="$(arg base_pose_topic)" />
        <param name="topo_position_service" type="string" value="$(arg topo_position_service)" />
    </node>

</launch>
